### DISCUSSION TITLE
Neper to Abaqus CAE

### DISCUSSION THREAD

**Discussion by Sofia-Y** [neperfepx/neper#131] [Q&A] [answered]
Hello @rquey ,

Is there any open-source python code that can create a CAE Abaqus file?
The only one I found was by Nicolò Grilli "Neper2CAE" (https://github.com/ngrilli/Neper2CAE). Unfortunately, I am not interested in columnar grains. I am interested in creating a CAE Abaqus file to get the same exact Neper model for more realistic cases. 

I appreciate your support on this platform. 
Thank you.

**Comment by rquey** [comment, marked as answer]
Not that I am aware of. Maybe someone out there already did it?

**Reply by Sofia-Y** [reply]
There are plenty of studies that have used Neper and Abaqus CAE. However, I did not find any of their python codes available as open-source. Hopefully, I will figure it out eventually. 

Thank you @rquey !

**Reply by rquey** [reply]
Yes, I know. It would be good if someone could show this to us.  We could archive it somewhere.

**Reply by ap264** [reply]
Hello Sofia-Y 

Have you solved your issue?

Thank you
